src/string.rs

```diff
@@ -125,8 +125,12 @@ impl Ord for IString {
         Ord::cmp(AsRef::<str>::as_ref(self), AsRef::<str>::as_ref(other))
     }
 }
```

Comment:
why doesnt cargo fmt test check for missing newlines inbetween items

Reply:
I've never seen rustfmt care about newlines between items. Regardless, I've added one.

Reply:
yes, it doesnt care, and thats what im complaining about lmao
you good tho